src/components/Sections/SectionChart/SectionPieChart.js

```diff
@@ -87,7 +57,11 @@ const SectionPieChart = ({ data, style, dimensions, legend, chartProperties = {}
   if (sortBy) {
     preparedData = orderBy(preparedData, sortBy.values, sortBy.orders);
   }
-
+  if (total > 0 && (chartProperties.showValues === undefined || chartProperties.showValues)) {
```

Comment:
```suggestion
  if (total > 0 && (chartProperties.showValues === true )) {
```

Reply:
i also would like to support backward compatibility (still show percentage labels)